apps/web/cypress/tests/notifications.spec.ts

```diff
@@ -24,4 +24,28 @@ describe('Notification Templates Screen', function () {
     cy.getByTestId('create-template-btn').should('not.be.disabled');
     cy.getByTestId('category-label').contains('General');
   });
+
+  it('when no workflow templates created it should show the page placeholder', function () {
+    cy.initializeSession({ noTemplates: true }).as('session');
+    cy.intercept('**/notification-templates**').as('notificationTemplates');
+    cy.visit('/templates');
+    cy.wait('@notificationTemplates');
+
+    cy.getByTestId('no-workflow-templates-placeholder').should('be.visible');
+    cy.getByTestId('create-workflow-tile').should('not.be.disabled');
+    cy.getByTestId('try-digest-playground-tile').should('not.be.disabled');
+  });
+
+  it('when clicking on create workflow it should redirect to create template page', function () {
+    cy.initializeSession({ noTemplates: true }).as('session');
+    cy.intercept('**/notification-templates**').as('notificationTemplates');
+    cy.visit('/templates');
+    cy.wait('@notificationTemplates');
+
+    cy.getByTestId('no-workflow-templates-placeholder').should('be.visible');
+    cy.getByTestId('create-workflow-tile').should('not.be.disabled');
+    cy.getByTestId('create-workflow-tile').click();
+
+    cy.url().should('include', '/templates/create');
+  });
```

Comment:
for now only two tests...
I'll add the test for the `Try Digest Workflow` tile a little later when it will point to the appropriate page